toolchain/sem_ir/ids.h

```diff
@@ -192,27 +192,17 @@ struct CompileTimeBindIndex : public IndexBase<CompileTimeBindIndex> {
   using IndexBase::IndexBase;
 };
 
-// The index of a runtime parameter in a function. These are allocated
+// The index of a `Call` parameter in a function. These are allocated
 // sequentially, left-to-right, to the function parameters that will have
-// arguments passed to them at runtime. In a `call` instruction, a runtime
+// arguments passed to them at runtime. In a `Call` instruction, a runtime
 // argument will have the position in the argument list corresponding to its
-// runtime parameter index.
-// TODO: Rename this to CallParamIndex, for consistency with the "`Call`
-// parameters" terminology in EntityWithParamsBase.
-struct RuntimeParamIndex : public IndexBase<RuntimeParamIndex> {
+// `Call` parameter index.
+struct CallParamIndex : public IndexBase<CallParamIndex> {
   static constexpr llvm::StringLiteral Label = "runtime_param";
```

Comment:
Do you want to change this here? Or maybe useful to do in a follow-up patch to isolate the noisy churn?

(If in a follow-up, leave a TODO for this PR?)

Reply:
Good catch, yes, we should change this. This PR was already supposed to isolate the noisy churn of dropping all those `value_param_pattern`s (although I admit there's been some scope creep), so it seems simpler to take care of this now.

Reply:
So, having seen this, I think it would be better to do this in a follow-up PR. Changing this changes essentially every test file, where-as without this there was a fairly reasonable subset of tests really impacted by the change.

I'm approving, but I'd suggest undoing this and leaving a TODO and having a PR that does *just* this rename so that it's a bit more isolated. Otherwise, for example, I don't think I could find the interesting diagnostic change in the output.

Reply:
OK, done.